ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { Telemetry } from 'types/config.proto';
+
+import { selectors } from 'reducers';
+
+const useAnalytics = () => {
+    const telemetry = useSelector(selectors.publicConfigTelemetrySelector);
+    const { enabled: telemetryEnabled } = telemetry || ({} as Telemetry);
```

Comment:
Minor point of naming in comparison to the companion pull request where you followed our more common convention `isTelemetryEnabled` for a boolean variable.